package.json

```diff
@@ -0,0 +1,46 @@
+{
+    "name": "wl-tracking",
+    "author": "Workleap",
+    "description": "This package allows you to add basic tracking capabilities to your applications",
+    "private": true,
+    "type": "module",
+    "license": "Apache-2.0",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/workleap/wl-tracking.git"
```

Comment:
Add a `directory` field?

Ex: `"directory": "packages/mixpanel"`

Reply:
only in the package.json of that folder. This is the monorepo root, no need for directly